src/uform.py

```diff
@@ -33,8 +33,6 @@ def __init__(
         )
         self.backbone_type = backbone_type
         self.pooling = pooling
-        self.dim = dim
```

Comment:
Why is this part gone? The goal was to add it everywhere to be able to infer the properties of the model.